### PR TITLE
New package: bencoepp.kommod version 1.0.0.0

### PR DESCRIPTION
**Pull request by bencoepp**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/117615)

### CODE REVIEW

File: manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: bencoepp.kommod
+PackageVersion: 1.0.0.0
```
Comment (Trenly) [inline manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.installer.yaml:5]:
```suggestion
PackageVersion: 0.0.0.1
```
This should match what the package reports as it's own version
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> kommod -v
0.0.0.1
```

File: manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.locale.de.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: bencoepp.kommod
+PackageVersion: 1.0.0.0
```
Comment (Trenly) [inline manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.locale.de.yaml:5]:
```suggestion
PackageVersion: 0.0.0.1
```
This should match what the package reports as it's own version
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> kommod -v
0.0.0.1
```

File: manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: bencoepp.kommod
+PackageVersion: 1.0.0.0
```
Comment (Trenly) [inline manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.yaml:5]:
```suggestion
PackageVersion: 0.0.0.1
```
This should match what the package reports as it's own version
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> kommod -v
0.0.0.1
```

File: manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: bencoepp.kommod
+PackageVersion: 1.0.0.0
+InstallerType: portable
+Commands:
+- kommod
+Installers:
+- InstallerUrl: https://github.com/BenCoepp/kommod/releases/download/latest/kommod.exe
```
Comment (Trenly) [inline manifests/b/bencoepp/kommod/1.0.0.0/bencoepp.kommod.installer.yaml:10]: This should point to a static release instead of latest

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by BenCoepp** [comment]
> @BenCoepp the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree company="Ben Coepp Publishing"

**Comment by stephengillie** [comment]
Hi @BenCoepp,

>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\b\bencoepp\kommod\1.0.0.0', expected 'manifests\b\bencoepp\kommod\0.0.0.1'

This error is because the file path has to match the PackageVersion. All 3 folders will need to be moved from the `1.0.0.0` folder into a new `0.0.0.1` folder. This can be done in your file system or in GitHub. I'd be happy to help with this if you like. 

Also, all 3 files have to have a matching PackageVersion, so the [locale file](https://github.com/microsoft/winget-pkgs/pull/117615/files#diff-608aed75f9ad5b29310b6d2339a10ca1edbb9264fa87ec2597e74436f46239ae) will need this updated as [suggested by Trenly](https://github.com/microsoft/winget-pkgs/pull/117615/files#r1301708002).

**Pull request closed without merging**